styles.css

```diff
@@ -0,0 +1,100 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #1a1a1a;
+}
+
+#myCarousel {
+  margin-top: 60px;
+}
+
+#carouselTag {
+  text-align: center;
+  margin: 0 auto;
+  margin-top: 5px;
+}
+
+#carouselTag > a {
+  color: #333;
+  margin: 5px 3px;
+  font-size: 14px;
+}
+
+#carouselTag > a:hover {
+  color: #83fe00;
+  text-decoration: none;
+}
+/* ---Media Queries------ */
+
+@media (max-width: 549px) {
+  #myCarousel .item { height: 310px; }
+
+  #slide1 {
+    background: url('images/carousel_xsmall_01_run_opt.png') top center no-repeat;
```

Comment:
See if you can extract the background positioning into its own class.
https://developer.mozilla.org/en-US/docs/Web/CSS/background?v=control